File: Pelikula.API/Model/Anketa/AnketaOdgovorResponse.cs
namespace Pelikula.API.Model.Anketa
{
    public class AnketaOdgovorResponse
    {
        public int Id { get; set; }
        public string Odgovor { get; set; }
        public int RedniBroj { get; set; }
        public int UkupnoIzabrano { get; set; }
    }
}

File: Pelikula.MOBILE/lib/model/anketa_odgovor_response.dart
class AnketaOdgovorResponse {
  int? id;
  String? odgovor;
  int? redniBroj;
  int? ukupnoIzabrano;

  AnketaOdgovorResponse({
    this.id,
    this.odgovor,
    this.redniBroj,
    this.ukupnoIzabrano,
  });

  factory AnketaOdgovorResponse.fromJson(Map<String, dynamic> json) {
    return AnketaOdgovorResponse(
      id: json['id'] as int,
      odgovor: json['odgovor'] as String,
      redniBroj: json['redniBroj'] as int,
      ukupnoIzabrano: json['ukupnoIzabrano'] as int,
    );
  }

  Map<String, dynamic> toJson() => {
        "id": id,
        "ododgovor": odgovor,
        "reredniBroj": redniBroj,
        "ukupnoIzabrano": ukupnoIzabrano,
      };
}
